api-reference/v1.0/api/user_get.md

```diff
@@ -24,9 +24,12 @@ This method supports the [OData Query Parameters](http://developer.microsoft.com
 Do not supply a request body for this method.
 ## Response
 If successful, this method returns a `200 OK` response code and [user](../resources/user.md) object in the response body.
-## Example
+
+## Examples
+
+By default, only a limited set of properties are returned (_businessPhones, displayName, givenName, id, jobTitle, mail, mobilePhone, officeLocation, preferredLanguage, surname, userPrincipalName_). This example illustrates the default request and response.
+
 ##### Request
```

Comment:
I wonder whether we should have another level (### level) heading to make it clear there are 2 examples:
### Example 1:  Standard users request
...
### Example 2: Users request user $select